Gemfile

```diff
@@ -8,10 +8,6 @@ group :test do
   gem "codeclimate-test-reporter", "~> 1.0.0", :require => false
 end
 
-unless dependencies.detect { |d| d.name == "xclarity_client" }
-  gem "xclarity_client"
-end
-
```

Comment:
that dependency is already in [manageiq-providers-lenovo.gemspec](https://github.com/ManageIQ/manageiq-providers-lenovo/blob/master/manageiq-providers-lenovo.gemspec#L16)

@juliancheal do you know if this can be removed or why it should stay?